MediaBrowser.Api/ApiEntryPoint.cs

```diff
@@ -610,9 +610,8 @@ private async Task KillTranscodingJob(TranscodingJob job, bool closeLiveStream,
                             process.Kill();
                         }
                     }
-                    catch (Exception ex)
+                    catch (InvalidOperationException)
                     {
-                        Logger.LogError(ex, "Error killing transcoding job for {Path}", job.Path);
```

Comment:
I would actually like a comment here explaining exactly _why_ this catch is empty, so we won't add it back later